server/model/mongoModel.js

```diff
@@ -6,6 +6,13 @@ const getListing = (id) => {
   return connection.db.collection('listings').find({id: id}).toArray().then((results) => {
     return results[0]
   })
+  // return new Promise((resolve, reject) => {
+  //   let cursor = connection.db.collection('listings').find({id: id});
+  //   let results;
+  //   cursor.on('data', (doc) => {
+  //     resolve(doc);
+  //   });
+  // })
```

Comment:
Just flagging this block of commented-out code so you can see if you still want to keep it.